rebar.config

```
@@ -10,17 +10,17 @@
 
 {deps, [
     {clique, ".*", {git, "https://github.com/helium/clique.git", {branch, "develop"}}},
-    {lager, "3.7.0"},
-    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {branch, "v2.8.0"}}},
+    {lager, "3.9.2"},
+    {jsx, "3.1.0"},
     {kvc, {git, "https://github.com/etrepum/kvc", {tag, "v1.7.0"}}},
-    {observer_cli, "1.5.0"},
+    {observer_cli, "1.7.1"},
     {hackney, "1.15.2"},
-    {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "v1.1.1"}}},
+    {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "v1.4.6"}}},
```

```suggestion
    {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "1.4.7"}}},
```
1.4.7 is the latest, and they dropped the `v` from the tags